style.css

```diff
@@ -0,0 +1,101 @@
+@media only screen and (max-width: 600px) {
```

Comment:
- [ ] Remember that you are working on a mobile-first approach, which means that your design has to look as the design until 768px of width, please modify this value so it follows the design until the desktop version


![image](https://user-images.githubusercontent.com/93630700/204954449-241b7dfd-94ee-4157-85f3-ff4a9d60f7f0.png)